notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1285_top_end_assembly_vibration.ipynb

```diff
@@ -109,9 +109,9 @@
     "# start = Time(\"2024-03-10T18:00:00Z\", scale=\"utc\", format=\"isot\")\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        long_name="SST M2 Surrogate",
</pre><p>Trivial, but what is the meaning of SST ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/140/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Reply:
I added a comment to answer this question. SST stands for Simonyi Survey Telescope.